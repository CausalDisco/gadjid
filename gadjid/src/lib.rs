// SPDX-License-Identifier: MPL-2.0
#![warn(missing_docs)]
//! gadjid -  Graph Adjustment Identification Distance library
mod ascending_list_utils;
mod graph_loading;
pub mod graph_operations;
mod partially_directed_acyclic_graph;

pub use graph_loading::constructor::EdgelistIterator;
pub use partially_directed_acyclic_graph::LoadError;
pub use partially_directed_acyclic_graph::PDAG;
