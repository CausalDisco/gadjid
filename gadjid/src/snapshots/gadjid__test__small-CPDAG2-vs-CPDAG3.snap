---
source: src/lib.rs
expression: "test(&format!(\"200{:0>2}.CPDAG-10\", true_id),\n    &format!(\"200{:0>2}.CPDAG-10\", guess_id))"
---
g_true: 20002.CPDAG-10
g_guess: 20003.CPDAG-10
ancestor_aid:
  - 0.37777777777777777
  - 34
oset_aid:
  - 0.3888888888888889
  - 35
parent_aid:
  - 0.5555555555555556
  - 50
shd:
  - 0.4444444444444444
  - 20
T:
  - 7
NAM:
  - 1
  - 4
  - 8
  - 9
Y: 1
random_adj:
  - 8
  - 9
proper_ancestors:
  - 1
  - 2
  - 3
  - 9
oset:
  - 2
  - 3
  - 7
  - 9
possible_descendants:
  - 1
  - 4
  - 7
  - 8
  - 9
parent_adjustment_NVA:
  - 0
  - 2
  - 3
  - 4
  - 5
  - 6
  - 8
  - 9
ancestor_adjustment_NVA:
  - 0
  - 2
  - 3
  - 4
  - 5
  - 6
  - 8
  - 9
nondescendant_adjustment_NVA:
  - 0
  - 2
  - 3
  - 4
  - 5
  - 6
  - 9
oset_adjustment_NVA:
  - 0
  - 2
  - 3
  - 4
  - 5
  - 6
  - 7
  - 8
  - 9
empty_adjustment_NVA:
  - 0
  - 2
  - 3
  - 4
  - 5
  - 6
  - 8
  - 9
random_Z_adjustment_NVA:
  - 0
  - 3
  - 4
  - 5
  - 6
  - 8
  - 9
